HTML/EN/settings/server/behavior.html

```diff
@@ -43,6 +43,11 @@
 			[% END %]
 		[% END %]
 
+		[% WRAPPER settingGroup title="" desc=""%]
+			<div class="prefDesc">[% "SETUP_ALBUMLINKROLES_DESC" | string %]<br />[% "SETUP_ALBUMLINKROLES_DESC2" | string %]</div>
+			<input type="text"class="stdedit" name="pref_albumLinkRoles" id="albumLinkRoles" value="[% prefs.pref_albumLinkRoles | html %]" size="40" >
```

Comment:
Couldn't we have this as a list of checkboxes? Looks odd to have checkboxes for one set of options, and a text input list for another one.